lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -392,6 +392,25 @@ static bool isBound(hw::HWModuleLike op, hw::InstanceGraph &instanceGraph) {
   });
 }
 
+// Add any existing bindings to the bind table.
+static void addBinds(hw::HWModuleLike op, hw::InstanceGraph &instanceGraph,
+                     SmallVectorImpl<BindOp> &existingBinds,
+                     BindTable &bindTable) {
+  auto *node = instanceGraph.lookup(op);
+  for (hw::InstanceRecord *use : node->uses()) {
+    auto inst = dyn_cast_or_null<hw::InstanceOp>(use->getInstance());
+    if (!inst)
+      continue;
+    if (!inst->hasAttr("doNotPrint") || !inst.getInnerSym().has_value())
+      continue;
+
+    auto parent = inst->getParentOfType<hw::HWModuleOp>();
+    for (auto bind : existingBinds)
```

Comment:
Can we use map for `existingBinds`? Practically I don't think it will cause a regression but it would be more efficient.

Reply:
This got me thinking... we have the information we need to just add the existing binds to the bind table directly.